Problem(s) with 0.8.0?

When I pull down master, yarn, yarn build, npm link, and try to run microbundle, I get:
```
/usr/local/bin/microbundle: line 1: syntax error near unexpected token `ex'
/usr/local/bin/microbundle: line 1: `function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }'
```
I noticed `dist/cli.js` is missing `#!/usr/bin/env node`. Adding it and trying to run microbundle again, I get: 
```
(babel plugin) Error: Cannot find module '@babel/plugin-syntax-jsx' from '/Users/iwalter/ianwalter/vuex-reset'
Error: Cannot find module '@babel/plugin-syntax-jsx' from '/Users/iwalter/ianwalter/vuex-reset'
    at Function.module.exports [as sync] (/Users/iwalter/microbundle/node_modules/resolve/lib/sync.js:43:15)
    at resolveStandardizedName (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at alias (/Users/iwalter/microbundle/node_modules/@babel/core/lib/config/config-descriptors.js:63:49)
```
I don't have `@babel/plugin-syntax-jsx` in my project's babel config and I get the same error in a project without a babel config so there seems to be an issue with how babel was added in #252? Or am I missing something?

--- a/babel.config.js
+++ b/babel.config.js
@@ -3,21 +3,12 @@ module.exports = {
 		[
 			'@babel/preset-env',
 			{
-				exclude: [
-					'transform-async-to-generator',
-					'proposal-async-generator-functions',
-					'transform-regenerator',
-				],
 				loose: true,
 				targets: {
-					chrome: '58',
-					ie: '11',
+					node: 'current',
 				},
 			},
 		],
 	],
-	plugins: [
-		['babel-plugin-transform-async-to-promises', { inlineHelpers: true }],
-		'@babel/plugin-syntax-jsx',
-	],
+	plugins: ['@babel/plugin-syntax-jsx'],
 };

--- a/package.json
+++ b/package.json
@@ -90,7 +90,7 @@
     "husky": "^1.1.2",
     "jest": "^23.6.0",
     "lint-staged": "^8.0.0",
-    "prettier": "^1.13.0",
+    "prettier": "^1.15.3",
     "regenerator-runtime": "^0.12.1",
     "rimraf": "^2.6.2",
     "shell-quote": "^1.6.1",

--- a/src/cli.js
+++ b/src/cli.js
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
+
 import microbundle from '.';
 import prog from './prog';
 import { stdout } from './utils';

--- a/src/index.js
+++ b/src/index.js
@@ -106,8 +106,8 @@ export default async function microbundle(options) {
 				(await isFile(resolve(cwd, filename + '.ts')))
 					? '.ts'
 					: (await isFile(resolve(cwd, filename + '.tsx')))
-					? '.tsx'
-					: '.js'
+						? '.tsx'
+						: '.js'
 			}`,
 		);
 
@@ -117,10 +117,9 @@ export default async function microbundle(options) {
 			options.entries && options.entries.length
 				? options.entries
 				: (options.pkg.source && resolve(cwd, options.pkg.source)) ||
-						((await isDir(resolve(cwd, 'src'))) &&
-							(await jsOrTs('src/index'))) ||
-						(await jsOrTs('index')) ||
-						options.pkg.module,
+				  ((await isDir(resolve(cwd, 'src'))) && (await jsOrTs('src/index'))) ||
+				  (await jsOrTs('index')) ||
+				  options.pkg.module,
 		)
 		.map(file => glob(file))
 		.forEach(file => options.input.push(...file));
@@ -377,12 +376,15 @@ function createConfig(options, entry, format, writeMeta) {
 						babelrc: false,
 						exclude: 'node_modules/**',
 						plugins: [
-							'@babel/plugin-syntax-jsx',
+							require.resolve('@babel/plugin-syntax-jsx'),
 							[
-								'babel-plugin-transform-async-to-promises',
+								require.resolve('babel-plugin-transform-async-to-promises'),
 								{ inlineHelpers: true, externalHelpers: true },
 							],
-							['@babel/plugin-proposal-class-properties', { loose: true }],
+							[
+								require.resolve('@babel/plugin-proposal-class-properties'),
+								{ loose: true },
+							],
 						],
 					}),
 					{

--- a/src/index.js
+++ b/src/index.js
@@ -106,8 +106,8 @@ export default async function microbundle(options) {
 				(await isFile(resolve(cwd, filename + '.ts')))
 					? '.ts'
 					: (await isFile(resolve(cwd, filename + '.tsx')))
-					? '.tsx'
-					: '.js'
+						? '.tsx'
+						: '.js'
 			}`,
 		);
 
@@ -117,10 +117,9 @@ export default async function microbundle(options) {
 			options.entries && options.entries.length
 				? options.entries
 				: (options.pkg.source && resolve(cwd, options.pkg.source)) ||
-						((await isDir(resolve(cwd, 'src'))) &&
-							(await jsOrTs('src/index'))) ||
-						(await jsOrTs('index')) ||
-						options.pkg.module,
+				  ((await isDir(resolve(cwd, 'src'))) && (await jsOrTs('src/index'))) ||
+				  (await jsOrTs('index')) ||
+				  options.pkg.module,
 		)
 		.map(file => glob(file))
 		.forEach(file => options.input.push(...file));
@@ -319,6 +318,8 @@ function createConfig(options, entry, format, writeMeta) {
 	}
 	loadNameCache();
 
+	let shebang;
+
 	let config = {
 		inputOptions: {
 			input: exportType ? resolve(__dirname, '../src/lib/__entry__.js') : entry,
@@ -383,6 +384,28 @@ function createConfig(options, entry, format, writeMeta) {
 							['@babel/plugin-proposal-class-properties', { loose: true }],
 						],
 					}),
+					{
+						// Custom plugin that removes shebang from code because newer
+						// versions of bublé bundle their own private version of `acorn`
+						// and I don't know a way to patch in the option `allowHashBang`
+						// to acorn.
+						// See: https://github.com/Rich-Harris/buble/pull/165
+						transform(code) {
+							let reg = /^#!(.*)/;
+							let match = code.match(reg);
+
+							if (match !== null) {
+								shebang = '#!' + match[0];
+							}
+
+							code = code.replace(reg, '');
+
+							return {
+								code,
+								map: null,
+							};
+						},
+					},
 					buble({
 						exclude: 'node_modules/**',
 						jsx: options.jsx || 'h',
@@ -459,7 +482,9 @@ function createConfig(options, entry, format, writeMeta) {
 							config._code = code;
 						},
 					},
-					shebangPlugin(),
+					shebangPlugin({
+						shebang,
+					}),
 				)
 				.filter(Boolean),
 		},

--- a/src/index.js
+++ b/src/index.js
@@ -106,8 +106,8 @@ export default async function microbundle(options) {
 				(await isFile(resolve(cwd, filename + '.ts')))
 					? '.ts'
 					: (await isFile(resolve(cwd, filename + '.tsx')))
-						? '.tsx'
-						: '.js'
+					? '.tsx'
+					: '.js'
 			}`,
 		);
 
@@ -117,9 +117,10 @@ export default async function microbundle(options) {
 			options.entries && options.entries.length
 				? options.entries
 				: (options.pkg.source && resolve(cwd, options.pkg.source)) ||
-				  ((await isDir(resolve(cwd, 'src'))) && (await jsOrTs('src/index'))) ||
-				  (await jsOrTs('index')) ||
-				  options.pkg.module,
+						((await isDir(resolve(cwd, 'src'))) &&
+							(await jsOrTs('src/index'))) ||
+						(await jsOrTs('index')) ||
+						options.pkg.module,
 		)
 		.map(file => glob(file))
 		.forEach(file => options.input.push(...file));

--- a/test/fixtures/shebang/package.json
+++ b/test/fixtures/shebang/package.json
@@ -1,0 +1,3 @@
+{
+  "name": "shebang"
+}

--- a/test/fixtures/shebang/src/index.js
+++ b/test/fixtures/shebang/src/index.js
@@ -1,0 +1,5 @@
+#!/usr/bin/env node
+
+export function foo() {
+	return 'hello world';
+}

--- a/test/index.test.js
+++ b/test/index.test.js
@@ -88,4 +88,12 @@ describe('fixtures', () => {
 			).toMatchSnapshot();
 		});
 	});
+
+	it('should keep shebang', () => {
+		expect(
+			fs
+				.readFileSync(resolve(FIXTURES_DIR, 'shebang/dist/shebang.js'), 'utf8')
+				.startsWith('#!'),
+		).toEqual(true);
+	});
 });

